fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/cache/CommonTokenValuesCacheImpl.java

```diff
@@ -29,22 +30,28 @@
 
     private final ThreadLocal<LinkedHashMap<CommonTokenValue, Long>> commonTokenValues = new ThreadLocal<>();
 
+    // thread-local cache of canonicals
+    private final ThreadLocal<LinkedHashMap<String, Integer>> canonicalValues = new ThreadLocal<>();
+
     // The lru cache shared at the server level
     private final LRUCache<String, Integer> codeSystemsCache;
 
     // The lru cache shared at the server level
     private final LRUCache<CommonTokenValue, Long> tokenValuesCache;
 
+    // The lru cache shared at the server level
+    private final LRUCache<String, Integer> canonicalValuesCache;
 
     /**
      * Public constructor
      * @param sharedExternalSystemNameCacheSize
```

Comment:
Update params in comment

Reply:
ok...I think I did what you meant: e.g.    // The lru canonical values cache shared at the server level

Reply:
The comment was supposed to be for line 49 (Public constructor).  The line numbers must have shifted a little bit.